Source/OxyPlot/Series/BoxPlotItem.cs

```diff
@@ -36,12 +36,39 @@ public struct BoxPlotItem
             double upperWhisker,
             IList<double> outliers = null,
             object tag = null)
+            : this(x, lowerWhisker, boxBottom, median, double.NaN, boxTop, upperWhisker, outliers, tag)
+        {
+        }
+        
+        /// <summary>
+        /// Initializes a new instance of the <see cref="BoxPlotItem" /> struct.
+        /// </summary>
+        /// <param name="x">The x.</param>
+        /// <param name="lowerWhisker">The lower whisker.</param>
+        /// <param name="boxBottom">The box bottom.</param>
+        /// <param name="median">The median.</param>
+        /// <param name="mean">The mean.</param>
+        /// <param name="boxTop">The box top.</param>
+        /// <param name="upperWhisker">The upper whisker.</param>
+        /// <param name="outliers">The outliers.</param>
+        /// <param name="tag">The tag.</param>
+        public BoxPlotItem(
```

Comment:
it is difficult to maintain ctors with arguments - cannot change order or add/remove without making a breaking change. I prefer to use parameterless ctors and use initializers instead!